hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/schemas/GenesisSchema.java

```diff
@@ -273,11 +276,35 @@ private void createGenesisExchangeRate(
     // Creates and loads the network properties into state
 
     private void createGenesisNetworkProperties(
-            @NonNull final BootstrapConfig v,
+            @NonNull final BootstrapConfig bootstrapConfig,
             @NonNull final FilesConfig filesConfig,
-            @NonNull final WritableKVState<FileID, File> files) {
+            @NonNull final WritableKVState<FileID, File> files,
+            @NonNull final Configuration configuration) {
         logger.debug("Creating genesis network properties file");
-        // TBD Implement this method
+
+        // Get the set of network properties from configuration, and generate the file content to store in state.
+        List<Setting> settings = new ArrayList<>();
```

Comment:
`settings` is in deterministic order because `networkProperties` returns a `SortedMap` and all else follows; this is fine, but if instead the sorting was done here where the file is created the deterministic order would be clear right here and not more "remote" ...

Reply:
I like your suggestion. I would apply it immediately, but then I have to get all the sign-offs again (grrr). Let me address this in a follow up PR.